tools/src/wyvern/tools/lexing/WyvernLexer.x

```diff
@@ -253,7 +253,7 @@ import wyvern.tools.errors.ToolError;
     terminal Token cSquareBracket_t ::= /\]/ {: RESULT = token(RBRACK,lexeme); :};
 
  	terminal Token shortString_t ::= /(('([^'\n]|\\.|\\O[0-7])*')|("([^"\n]|\\.|\\O[0-7])*"))|(('([^']|\\.)*')|("([^"]|\\.)*"))/ {:
- 		RESULT = token(STRING_LITERAL,lexeme.substring(1,lexeme.length()-1));
+ 		RESULT = token(STRING_LITERAL,lexeme.substring(1,lexeme.length()-1).replace("\\n", "\n"));
```

Comment:
This solves the problem of introducing newlines, but makes it impossible to write \ in a string when followed by an n...plus all the other escapes are not implemented.

Would it be possible to use one of the solutions mentioned below, for example either using the Properties() library or the StringTokenizer() library?

http://stackoverflow.com/questions/1327355/is-there-a-java-function-which-parses-escaped-characters
http://stackoverflow.com/questions/3537706/how-to-unescape-a-java-string-literal-in-java